charts/ceph-csi-cephfs/templates/provisioner-clusterrole.yaml

```diff
@@ -53,12 +53,4 @@ rules:
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
 {{- end -}}
-{{- if .Values.topology.enabled }}
```

Comment:
please remove the topology section in value.yaml also?

Reply:
@Madhu-1 sure , done